prow/git/v2/publisher.go

```diff
@@ -28,14 +28,23 @@ type Publisher interface {
 	Commit(title, body string) error
 	// ForcePush runs `git push -f` to the publish remote
 	ForcePush(branch string) error
+	// PublishPush pushes the local state to the publish remote
+	PublishPush(branch string, force bool) error
```

Comment:
`PublishPush`? What about maybe `PushToFork`? As everywhere else, please use functional opts pattern for opts rather than requiring them to be set.

Reply:
I kept the same naming here, since there are two remotes, central and publish ones.

Reply:
same as what?

Reply:
Okay but this is still confusing because there is a `ForcePush` that pushes to the same fork as the `PublishPush` but the method doesn't have `Publish` in its name. Wouldn't it be better to name this just `Push` if the `PublishRemote` is the default so its consistent?

Reply:
I will take a look at what it takes to completely remove the `ForcePush`. From the first look, only the cherrypicker is using that.